backend/climateconnect_api/utility/email_setup.py

```diff
@@ -272,3 +285,50 @@ def remove_contact_from_list(contact_id, list_id):
         ]
     }
     mailjet_api.contact_managecontactslists.create(id=contact_id, data=data)
+
+
+def send_email_reminder_for_unread_notifications(
+    user: User,
+    user_notifications: List[UserNotification]
+):
+    total_notifications = user_notifications.count()
+    language_code = get_user_lang_code(user=user)
+    subject_by_language = {
+        "en": f"You have {total_notifications} unread notifications",
+        "de": f"Sie haben {total_notifications} ungelesene Benachrichtigungen"
```

Comment:
We usually use the less formal way to address people

```suggestion
        "de": f"Du hast {total_notifications} ungelesene Nachrichten"
```